claims-related/tests/single_source_csv/single_source_csv_comp.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+import os
+import argparse
+
+
+def printDf(df, f):
+    columnOrder = ['VERSION', 'Measure_ID',
+                   'DATA_ELEMENT_NAME',
+                   'CODING_SYSTEM',
+                   'CODE',
+                   'MODIFIER',
+                   'PLACE_OF_SERVICE',
+                   'AGE',
+                   'GENDER'
+                   ]
+    # to markdown is better but unfortunately not supported by current docker
+    # print(df[columnOrder].to_markdown(tablefmt="grid", index=False), file=f)
```

Comment:
we can remove this comment

Reply:
@panha-yorn  If we execute this locally we can use that comment assuming that we have the to_markdown function. These are off-hand scripts that are executed by user and not necessarily need to use the recommended mechanism to execute. The idea was to use to_markdown which is *much nicer* but I am not committing it because of compatibility issue.

Reply:
Done added a comment specifying usage